L/libboxer/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "libboxer"
+version = v"0.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/feenkcom/libboxer/archive/refs/tags/v$(version).tar.gz",
+                  "57d25e5339b0c7b1d8c8242484ce2c4baca05fbea035c8e2d8aab05573eed0c9"),
+]
+
+# Adapted from the justfile of the repo
+script = raw"""
+cd $WORKSPACE/srcdir/libboxer*
+cargo build --release
+install -Dvm 0755 "target/${rust_target}/release/libBoxer.${dlext}" "${libdir}/libBoxer.${dlext}"
```

Comment:
I can't possibly know what's the content of the directory, I'm away from computer and will be so for a while, but why don't you simply add some ls commands to see what's inside the directory? Also, I already suggested you on slack to use [github codespaces](https://docs.binarybuilder.org/stable/#Github-Codespaces) which is now free for everyone, with with which you can interactively debug the builds yourself instead of several small commits to get this working.

Reply:
Ah, good idea! Let me try!

Reply:
Rust didn't work out for me on codespaces either (for a different reason though) :sweat_smile: